index.html

```diff
@@ -305,12 +305,11 @@ <h2>
         <aside class="note">
           In the case the steps are run from a dedicated worker, then the
           <a>current global object</a> is a {{DedicatedWorkerGlobalScope}},
-          which means that the <a>obtain permission</a> steps can never be
-          <a>triggered by user activation</a>, meaning that in order to use
-          wake locks from a dedicated worker, prior permission needs to granted
-          from the <a>browsing context</a> who created it, e.g. the <a>browsing
-          context</a> that is exists in the {{DedicatedWorkerGlobalScope}}'s
-          [=WorkerGlobalScope/owner set=].
+          which means that it will never have [=transient activation=].
```

Comment:
Please consider the last point above for this para too.  Something like:
> If the current global object is a DedicatedWorkerGlobalScope, it won't have transient activation...

Reply:
I've reworded things a bit, please take another look.